src/Repeater/ServerRepeaterLauncher.ts

```diff
@@ -164,6 +164,9 @@ export class ServerRepeaterLauncher implements RepeaterLauncher {
     this.repeaterServer.on(RepeaterServerEvents.RECONNECTION_SUCCEEDED, () =>
       logger.log('The Repeater (%s) connected', this.info.version)
     );
+    this.repeaterServer.on(RepeaterServerEvents.DEPLOY, () =>
+      logger.log('The Repeater (%s) deployed', this.info.version)
+    );
```

Comment:
Why do you need to expose such a low-level specific message at the "base" log level? Why not simply move `The Repeater (%s) started` message to the `deployRepeater` method?

Reply:
And, probably there is a bug. `RepeaterServerEvents.DEPLOY` must be replaced with the `RepeaterServerEvents.DEPLOYED` everywhere.